### DISCUSSION TITLE
Hide certain columns in an HXGrid on mobile view port?

### DISCUSSION THREAD

**Discussion by bluisanajmr** [havit/Havit.Blazor#660] [Q&A] [answered]
Is it possible to hide certain columns in an HXGrid on mobile view port? In the example below I want the grid to dynamically hide all columns other than the Project name column when in a mobile view port size. 

![image](https://github.com/havit/Havit.Blazor/assets/135136836/e3a32a7a-8735-4e62-abd0-d1d6da1af4b1)

**Comment by crdo** [comment, marked as answer]
You can achieve this by giving the columns one of the [Bootstraps utility classes](https://getbootstrap.com/docs/5.3/utilities/display/) to `ItemCssClass` and `HeaderCssClass` parameters.

**Reply by bluisanajmr** [reply]
Thanks, 

ItemCssClass="d-none d-md-table-cell" HeaderCssClass="d-none d-md-table-cell"

worked like a charm.
